test/test.js

```diff
@@ -3,150 +3,160 @@ import { single, accumulator, multiple, effectiveOdds } from '../lib/main';
 const test = require('tape');
 
 test('single win bet is correct', (t) => {
-  const selection = { stake: 10, odds: { decimal: 5 }, terms: 0.25 };
+  const selections = { stake: 10, odds: { decimal: 5 }, terms: '1/4' };
```

Comment:
Plural naming of `selections` suggests a collection of selections but only one selection is being assigned and not in an array. Whats the motivation for naming the variable as plural?

Reply:
My bad, that was a reckless `:%s`